### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Please review my wishlist page which contains the required context.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Small PR
Clean code
Good to merge

File: src/Context/wishlistContext.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { createContext, useContext, useReducer } from "react";
+
+const WishlistContext = createContext();
+
+const wishFunction = (wishState, action) =>{
+    switch(action.type){
+        case "Add_to_wishlist" :
```
Comment (Jia-A) [inline src/Context/wishlistContext.js]: Good practice is using capital letters for cases like "ADD_TO_WISHLIST". You can change that later.

File: src/cave-components/Navbar/navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const Navbar = () =>{
+
+const { wishState } = useWishlist();
+const { cartState } = useCart();
+
+return (
+<div className="App">
+    <div class="container">
+        <nav class="nav-header justify-align">
```
Comment (Jia-A) [inline src/cave-components/Navbar/navbar.jsx]: Good use of semantic HTML.

File: src/public-css/navbar.css [hunk 3]
```diff
@@ -35,16 +35,30 @@
...
 
-.btn {
+.btn-login {
     margin: 10px;
-    padding: 0.7rem;
     min-width: 6.25rem;
     cursor: pointer;
+    border-radius: 1.8rem;
+    padding: 0.8rem 1rem;
+
```
Comment (Jia-A) [inline src/public-css/navbar.css]: remove extra spacing

File: src/public-css/navbar.css [hunk 4]
```diff
@@ -53,4 +67,8 @@
```
Comment (Jia-A) [inline src/public-css/navbar.css]: You might wanna check what is happening here.

**Review by Jia-A — APPROVED** [review]
- Good naming conventions.
- Good file and code structure.
- Approved by me to merge after small suggested changes.

File: src/Context/wishlistContext.js (hunk 1 above)
Comment (Megha-Joshi) [reply src/Context/wishlistContext.js]: Okay

File: src/public-css/navbar.css (hunk 3 above)
Comment (Megha-Joshi) [reply src/public-css/navbar.css]: Okay

File: src/public-css/navbar.css (hunk 4 above)
Comment (Megha-Joshi) [reply src/public-css/navbar.css]: Yeah sure! Thanks for pointing it out.

**Pull request merged**